src/lib/components/InputGroup/__tests__/InputGroup.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {
+  render,
+  within,
+} from '@testing-library/react';
+import { Button } from '../../Button';
+import { SelectField } from '../../SelectField';
+import { TextField } from '../../TextField';
+import { childrenEmptyPropTest } from '../../../../../tests/propTests/childrenEmptyPropTest';
+import { InputGroup } from '../InputGroup';
+
+const mandatoryProps = {
+  children: [
+    <TextField key="Text field" label="Text field" />,
+    <SelectField
+      key="Select field"
+      label="Select field"
+      options={[{
+        label: 'label',
+        value: 'value',
+      }]}
+    />,
+    <Button key="Button" label="Button" />,
+  ],
+  label: 'Input group label',
+};
+
+describe('rendering', () => {
+  it.each([
+    ...childrenEmptyPropTest,
+    [
+      {},
+      (rootElement) => {
+        expect(within(rootElement).getByText('Input group label'));
```

Comment:
I would prefer a dedicated test for `isLabelVisible: true`. Here we test two props in one test, while elsewhere we always test one prop per one test case.